1738778003931	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1738788804011	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1738788804187	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1738788804187	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1738788804188	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1738788804703	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1738788804703	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1738788804704	Sync.Service	DEBUG	User-Agent: Firefox/135.0 (Windows NT 10.0; Win64; x64) FxSync/1.137.0.20250130195129.desktop
1738788804704	Sync.Service	INFO	Starting sync at 2025-02-06 03:53:24 in browser session hZKhevHs7mU7
1738788804704	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1738788804706	Sync.Status	INFO	Resetting Status.
1738788804706	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1738788804706	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1738788804706	FirefoxAccounts	DEBUG	getOAuthToken enter
1738788804706	FirefoxAccounts	TRACE	getCachedToken returning cached token
1738788804706	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1738788804707	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1738788804707	Sync.SyncAuthManager	INFO	Getting sync key
1738788804707	FirefoxAccounts	DEBUG	getOAuthToken enter
1738788804707	FirefoxAccounts	TRACE	getCachedToken returning cached token
1738788804707	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1738788804707	Sync.SyncAuthManager	INFO	Getting a sync token from: https://token.services.mozilla.com/1.0/sync/1.5
1738788804707	Sync.SyncAuthManager	DEBUG	Getting a token using OAuth
1738788804707	Services.Common.TokenServerClient	DEBUG	Beginning OAuth token exchange: https://token.services.mozilla.com/1.0/sync/1.5
1738788804707	Services.Common.RESTRequest	DEBUG	GET request to https://token.services.mozilla.com/1.0/sync/1.5
1738788804710	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2147500036 - NS_ERROR_ABORT
1738788804710	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_ABORT\"  nsresult: \"0x80004004 (NS_ERROR_ABORT)\"  location: \"JS frame :: resource://services-common/rest.sys.mjs :: onStopRequest :: line 478\"  data: no]"}
1738788804711	Services.Common.RESTRequest	DEBUG	GET https://token.services.mozilla.com/1.0/sync/1.5 failed: 2147500036 - NS_ERROR_ABORT
1738788804712	Sync.SyncAuthManager	ERROR	Non-authentication error in _fetchTokenForUser: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.sys.mjs:28:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.sys.mjs:51:16
_tokenServerExchangeRequest@tokenserverclient.sys.mjs:235:13
1738788804712	Sync.Status	DEBUG	Status.login: success.login => error.login.reason.network
1738788804712	Sync.Status	DEBUG	Status.service: success.status_ok => error.login.failed
1738788804712	FirefoxAccounts	TRACE	Getting command keys: https://identity.mozilla.com/cmd/open-uri
1738788804712	Sync.SyncAuthManager	ERROR	Failed to fetch a token for authentication: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.sys.mjs:28:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.sys.mjs:51:16
_tokenServerExchangeRequest@tokenserverclient.sys.mjs:235:13
1738788804713	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1738788804713	FirefoxAccounts	TRACE	Getting command keys: https://identity.mozilla.com/cmd/close-uri/v1
1738788804729	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/266395372/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1738788804729	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1738788804729	Sync.Status	DEBUG	Status.service: error.login.failed => error.sync.failed
1738788804729	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1738788804730	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=reschedule)
1738788804730	Sync.Telemetry	INFO	Early submission of sync telemetry due to changed IDs/NodeType
1738788804734	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
